### PR TITLE
docs: add a note about setting file executability

### PR DESCRIPTION
**Pull request by ryanlraines**
## Description
Adds a note about setting file executability to docs
...

## Related Issue

Fixes #

Relates to #

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: site/src/content/docs/ref/components.mdx [hunk 1]
```diff
@@ -52,6 +52,7 @@ Files can be:
 - Relative paths to either a file or directory (from the `zarf.yaml` file)
 - A remote URL (http/https)
 - Verified using the `shasum` field for data integrity (optional and only available for files)
+- made executable by setting `Executable` boolean value [true|false]
```
Comment (lucasrod16) [inline site/src/content/docs/ref/components.mdx:55]: Small suggestion on capitalization consistency and wording

```suggestion
- Made executable by setting the `executable` field to `true`
```

**Review by lucasrod16 — CHANGES_REQUESTED** [review]

**Comment by lucasrod16** [comment]
[This commit](https://github.com/defenseunicorns/zarf/pull/2435/commits/fd1b4f77501b0b141d0eb592261d13b5b3b92f3e) will need to be signed first before this PR can be merged

**Comment by ryanlraines** [comment]
> [This commit](https://github.com/defenseunicorns/zarf/pull/2435/commits/fd1b4f77501b0b141d0eb592261d13b5b3b92f3e) will need to be signed first before this PR can be merged

oops, sorry. 😞 

Thanks for being patient and explaining things 🙏🏼

**Comment by lucasrod16** [comment]
https://docs.github.com/en/authentication/managing-commit-signature-verification/associating-an-email-with-your-gpg-key

> Your GPG key must be associated with a GitHub verified email that matches your committer identity.

@ryanlraines I'm going to close this PR because of the unsigned and unverified commits.

I recommend getting commit signing set up to where every commit shows `Verified` next to it in GitHub by testing in a personal repo first to ensure it is setup correctly. Once everything is setup correctly and you've read through our [contributing guide](https://docs.zarf.dev/contribute/contributor-guide/#developer-workflow), feel free to open a new PR.

**Pull request closed without merging**